devel/ansible/roles/hotness-dev/files/motd

```diff
@@ -1,14 +1,27 @@
 
 Welcome to The New Hotness development environment!
 
+To start development you need to do this first:
+
+* add your bugzilla credentials to ~/.fedmsg.d/hotness.py
+
+* acquire a valid Kerberos ticket by `kinit <fas-username>@FEDORAPROJECT.ORG`
```

Comment:
I recommend using the staging environment instead of production if that's feasible. Bodhi uses production and I've been meaning to switch it to staging to prevent any accidental connections to production.

Reply:
I want to only simulate the connection for development environment. But you are right about this.